### PR TITLE
Bug/rw audio settings

### LINKED ISSUE #76: Audio Settings BUG
When a sound setting is selected, it seems to be saved in the DB. But upon page reload, if refreshed on the timer page, the sound settings are not updating. But, when refreshed on the settings page, the settings seem to be loading correctly. Please inspect! 
* Upon reloading the page to the timer, the sound settings are not displaying in context in the devtools. 
* But, upon reloading the page to the settings page, the sound settings look correct in the devtools.

### PR DESCRIPTION
**Pull request by rwilliams659**
# Som Timer Pull Request Form

## Description
**Audio bug**
The original task of this branch was to fix a bug with audio settings not saving, seemingly when setting audio, navigating to timer view, refreshing the page, and navigating back to settings. However, this seemed to always work correctly, and I could not reproduce the bug ever in that instance. 

However, I was able to initially reproduce the bug in 2 instances: a) When running a new build & serving it up, going to settings, changing audio setting & refreshing the page - however, this would only be the case ONCE i.e. on that first navigation & refresh on settings after the build was served. This effect was not observed if any other pages were navigated to before serving, or any subsequent times trying to update & refresh on that same build. This effect was not observed on the localhost:3000 version of the app ever. and b) Also within the build context, when setting audio & immediately refreshing (as fast as possible). 

HOWEVER, since the back-end was broken and then fixed, I have not been able to reproduce the bug in any situation, including the 2 detailed above. At this time, I think it makes sense to close this issue. We can keep an eye on it & see if it re-emerges in future iterations. 

**BreakInterval input bug** 
On exploration of settings, I noticed that the break interval option in settings wasn't displaying correctly. For example, if the break interval is changed to 10, it will say 5 if you go back to settings later, even though context says 10. I was able to fix this by adding a default value to that input set to the value of breakInterval in context. Additionally, in SettingsContext, I actually changed our default values for work & break interval to strings & removed the "+" that was parsing those values when returned from the apiCall. I did this because I realized that whenever those values were being updated, they actually were strings in context, and strings worked perfectly fine for the compoundTimer to correctly count down (must be some type coercion going on?). Since the values need to be strings when saved to the database, it made more sense for us to just always store them as strings too, so we don't have to worry about ever changing the data type. 

**Audio file names**
Lastly, I updated audio file names to reflect how they were displayed to the user. 

## Type of change
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [X] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`npm test`

# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

resolve #76

### CODE REVIEW

File: src/Components/Settings/Settings.js [hunk 1]
```diff
@@ -64,8 +65,8 @@ const Settings = () => {
           data-testid='sound'
           value={settings.sound}
         >
-          <option value={'chordCliff'}>Reverb Splash</option>
-          <option value={'gong'}>Balinese Gong</option>
+          <option value={'reverbSplash'}>Reverb Splash</option>
+          <option value={'balineseGong'}>Balinese Gong</option>
```
Comment (sienna-kopf) [inline src/Components/Settings/Settings.js:69]: YES!!!!!

File: src/Components/Settings/Settings.test.js [hunk 2]
```diff
@@ -51,10 +51,10 @@ describe('Settings', () => {
 
     fireEvent.blur(workIntervalInput, { target: { value: '45' } })
     fireEvent.blur(breakIntervalInput, { target: { value: '10' } })
-    fireEvent.change(soundInput, { target: { value: 'gong' } })
+    fireEvent.change(soundInput, { target: { value: 'balineseGong' } })
 
     expect(workIntervalInput.value).toBe('45')
     expect(breakIntervalInput.value).toBe('10')
-    expect(soundInput.value).toBe('gong')
+    expect(soundInput.value).toBe('balineseGong')
```
Comment (sienna-kopf) [inline src/Components/Settings/Settings.test.js:58]: SO much sicker than a normal gong 😂

**Review by sienna-kopf — APPROVED** [review]
Thanks for the detail in this PR @rwilliams659

File: src/Components/Settings/Settings.js (hunk 1 above)
Comment (Abdeboskey) [reply src/Components/Settings/Settings.js:69]: Thank you so much for normalizing these names,  I should have done this up front and I really appreciate you taking the time to fix it.

File: src/Components/Settings/Settings.test.js (hunk 2 above)
Comment (Abdeboskey) [reply src/Components/Settings/Settings.test.js:58]: You know it.

**Review by Abdeboskey — APPROVED** [review]
I really appreciate you chasing these bugs, and I'm glad that after troubleshooting the back end they seemed to disappear? Thank you for your thorough documentation in the PR as well.

**Review by jkwest-93 — APPROVED** [review]
Great work!! And glad this bug turned into a non-bug!

**Pull request merged**